From a2d6e87b82de92b58ad7503bc3f8b7e07fa9a373 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 10 Jul 2020 18:19:11 -0700
Subject: [PATCH 1435/1852] staging: apf: small ifdef - else

commit 209ccc16267a0db7ff32f209cb41e1681d9d7ec3 from
https://github.com/Xilinx/linux-xlnx.git

The line never reaches without else. Guard it using ifdef - else.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/staging/apf/xlnk.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/staging/apf/xlnk.c b/drivers/staging/apf/xlnk.c
index a359c37dfaf7..3998f482a1ee 100644
--- a/drivers/staging/apf/xlnk.c
+++ b/drivers/staging/apf/xlnk.c
@@ -717,8 +717,9 @@ static int xlnk_dmasubmit_ioctl(struct file *filp, unsigned int code,
 			return -EFAULT;
 	}
 	return status;
-#endif
+#else
 	return -ENOMEM;
+#endif
 }
 
 static int xlnk_dmawait_ioctl(struct file *filp,
-- 
2.31.1

